docs/DeveloperGuide.md

```diff
@@ -155,6 +155,54 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### FindCommonTiming feature
+This section details how the  'findCommonTiming' command is implemented. This command allows the user to 
+get the common free timings of contacts who share the same tag. The timings that the contacts are free
```

Comment:
DG does not have a line limit, so sentences should be part of the same line. The same for other parts of the DG.

Reply:
Alright I've amended this. Thanks!